operator/pkg/remote/kyma_synchronization_context.go

```diff
@@ -286,3 +290,24 @@ func GetRemoteObjectKey(kyma *v1alpha1.Kyma) client.ObjectKey {
 	}
 	return client.ObjectKey{Namespace: namespace, Name: name}
 }
+
+func (c *KymaSynchronizationContext) InsertWatcherLabels(ctx context.Context, remoteKyma *v1alpha1.Kyma) error {
+	recorder := adapter.RecorderFromContext(ctx)
+
+	ownedByValue := fmt.Sprintf("%s__%s", c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
```

Comment:
```suggestion
	ownedByValue := fmt.Sprintf("%s.%s", c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
```

Reply:
We can discuss on the format, but for now this is the format understood by the watcher, for parsing labels